### DISCUSSION TITLE
How to generate “USER-DEEPMD”

### DISCUSSION THREAD

**Discussion by Epikurz** [deepmodeling/deepmd-kit#4413] [Q&A]
In the document of deepmd-kit， there is description of install deepmd-lammps in built-in mode. It seems that users can make a USER-DEEPMD file in deepmd-kit/source/build with the command "make lammps". However there is no makefile and this command is invalid. How can I do the things in document to generate a USER-DEEPMD file? Thanks!

**Comment by njzjz** [comment]
Makefile is generated by CMake.
